src/main/resources/templates/leaderboard_lecturer.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="pl" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <meta charset="UTF-8"/>
+  <title>UPeL3.0</title>
+
+  <link th:rel="stylesheet" th:href="@{/webjars/bootstrap/4.0.0-2/css/bootstrap.min.css} "/>
+  <link rel="shortcut icon" href="https://upel2.cel.agh.edu.pl/wiet/theme/image.php/celupel/theme/1619178409/favicon" />
+</head>
+<body style="background-color: #ecf1fc;">
+
+<div th:replace="fragments/header :: header"></div>
+
+<section class="container py-3">
+  <header class="d-flex flex-wrap align-items-center justify-content-center justify-content-md-between pt-4 pb-2 mb-0" style="border-top: 3px solid #343a40;">
+    <h2 class="my-0">leader</h2>
+
+    <div class="text-end">
+      <a type="button" class="btn btn-primary" th:href="|course?id=${course.getId()}|">Powrót do kursu</a>
+    </div>
+  </header>
+  <nav class="border-bottom mt-0 pt-1 mb-3" aria-label="breadcrumb">
+    <ol class="breadcrumb" style="background-color: #ecf1fc">
+      <li class="breadcrumb-item"><a href="/index">Kursy</a></li>
+      <li class="breadcrumb-item"><a th:href="|/course?id=${course.getId()}|" th:text="${course.getName()}">Nazwa kursu</a></li>
+      <li class="breadcrumb-item active" aria-current="page">Leaderboard</li>
+    </ol>
+  </nav>
+  <p class="pb-3" style="border-bottom: 3px solid #343a40;">Tutaj możesz sprawdzić leaderboard kursu.</p>
+</section>
+
+<section class="container py-3">
+  <table class="table table-hover table-bordered bg-light">
+    <thead>
+    <tr>
+      <th scope="col">Numer</th>
+      <th scope="col">Student</th>
+      <th scope="col">Suma punktów</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr th:each="student : ${studentGrades}" th:class="${studentStat.count}">
```

Comment:
I'd add `th:style="${student.key} >= ${course.getPassValue()} ? 'background-color: #d0faa3;'"` to highlight green the ones that have already the course passed